src/store/links.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios');
+const cheerio = require('cheerio');
+
```

Comment:
the link.js shouldn't be in store directory .. it should with the rest of assets we are using .. like ' 	xml2json.min.js' at [src](https://github.com/chingu-voyage5/Geckos-Team-34/tree/master/src) or we can create a separate directory called 'libs' for both of them and any other thing we use .. 
we should look into that .. 
anyway please move this file to 'src'

Reply:
I have moved it into src/dist. Would that be fine?

Reply:
Yes .. Change  the variables names, add goodreads support and push it to your branch ..